recirq/qml_lfe/run_config.py

```diff
@@ -59,6 +63,32 @@ def execute_batch(
     return [x[0] for x in res]
 
 
+def execute_sweep(
+    circuit: cirq.Circuit, params: List[Dict[str, float]], use_engine: bool
+) -> List[cirq.Result]:
+    """Use either simulator or engine to run a one rep sweep on a circuit.
+
+    Args:
+        circuit: the `cirq.Circuit` with symbols to execute.
+        params: The list of parameters to place in the symbols.
+        use_engine: Whether or not to make use of quantum engine and the
+            weber processor. Note this requires the GOOGLE_CLOUD_PROJECT
+            environment variable to be set, along with the required cloud
+            permissions.
+    Returns:
+        List of results to execution.
+    """
+    runner = None
+    if use_engine:
+        runner = cirq_google.get_engine_sampler(
```

Comment:
why not pass around `runner` instead of a `use_engine` flag?

Reply:
I guess I could do that. I'll look into it in a follow on PR.